finetune/adapter_v2.py

```diff
@@ -46,7 +47,7 @@
 num_epochs = 5
 max_iters = num_epochs * (epoch_size // micro_batch_size) // devices
 weight_decay = 0.02
-warmup_steps = 2 * (epoch_size // micro_batch_size) // devices // gradient_accumulation_iters  # 2 epochs
+warmup_steps = math.ceil(2 * (epoch_size // micro_batch_size) // devices // gradient_accumulation_iters)  # 2 epochs
```

Comment:
@Chasapas Sorry for the naive question, what's the practical value here? The divisions here are all without remainder, so the result is never a value that needs to be rounded.